src/views/RequestItem.vue

```diff
@@ -127,19 +148,39 @@ export default defineComponent({
         resolveTime.value = formatDate(resPacketData.resolveTime)
         requestTimeRange.value = formatDateDifference(resPacketData.requestTime)
         resolveTimeRange.value = formatDateDifference(resPacketData.resolveTime)
-        requestCalldata.value = obiCoin.decode(reqPacketData.calldata)
+        requestCalldata.value = await _decodeCallData(reqPacketData.calldata)
 
-        if (requestStatus.value == 1) {
-          requestRates.value = obiRates.decode(resPacketData.result)
+        if (requestStatus.value === ResolveStatus.RESOLVE_STATUS_SUCCESS) {
```

Comment:
create computed property and replace in all component

Reply:
done